addons/stock/views/stock_picking_views.xml

```diff
@@ -378,7 +378,7 @@
                     <field name="lot_id" groups="stock.group_production_lot"/>
                     <filter name="to_do_transfers" string="To Do" domain="['&amp;',('user_id', 'in', [uid, False]),('state','not in',['done','cancel'])]"/>
                     <filter name="my_transfers" string="My Transfers" domain="[('user_id', '=', uid)]"/>
-                    <filter string="Starred" name="starred" domain="[('priority', '=', '1')]"/>
+                    <filter string="Starred" name="starred" domain="[('priority', '=', 1)]"/>
```

Comment:
shouldn't this be like so:
```suggestion
                    <filter string="Starred" name="starred" domain="[('is_favorite', '=', 1)]"/>
```

Reply:
Hello @pkyriakou ,
changes are done :)

Thankyou for the suggestion.